packages/frameworks-express/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
See https://github.com/nextauthjs/next-auth/blob/main/packages/core/tsconfig.json to match this file as closely as possible